### PR TITLE
Simtext

### PR DESCRIPTION
**Pull request by dlal-group**
Hi galaxy team, I created some training material for the SimText toolset that I would like to add to the galaxy training website. 

@bgruening Because you have been reviewing the tools before it might make sense that you have a look at it (if you have a bit of time). Thank you!

### CODE REVIEW

File: topics/text-mining/docker/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+#
+# to run image:
+#    docker run -p "8080:80" -t text-mining
+#    use -d to automatically dowload the datalibraries in the container
+
+FROM bgruening/galaxy-stable:latest
+
+MAINTAINER Galaxy Training Material
+
+ENV GALAXY_CONFIG_BRAND "GTN: Title of the topic"
```
Comment (hexylena) [inline topics/text-mining/docker/Dockerfile:15]:
```suggestion
ENV GALAXY_CONFIG_BRAND "GTN: Text Mining"
```

**Comment by shiltemann** [comment]
Just a general question: do you anticipate enough text-mining tutorials to justify having its own topic? Or should we add it to e.g. the "Statistics & Machine Learning" topic?

**Comment by dlal-group** [comment]
@shiltemann I do not anticipate many tutorials in the near future, so I think your suggestion to add it to "Statistics & Machine Learning" for now makes sense. Should I do the change in this PR or open a new one for it? 
Also, do you know what the problem in the "Lint and Build GTN / build-site" is? To me, it looks like a problem with the snippets but I think they have the same syntax as in other tutorials. 
Thank you so much!

**Comment by shiltemann** [comment]
@dlal-group sounds good, it would be great if you could do the moving in this PR

About the snippets: yes, we very recently overhauled how snippets work, I will update your branch to the new snippet syntax, so don't worry about the test for now :+1:

And above all: thanks for your contribution!

File: topics/statistics/tutorials/text-mining_simtext/tutorial.md [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+3. Calculation of the adjusted rand index (similarity between text-based grouping and the pre-existing disorder categories)
+4. Table with terms and their frequency among the genes
+
+> ### {% icon hands_on %} Hands-on: Explore data interactively
+>
+> 1. Run {% tool [interactive_tool_simtext_app](interactive_tool_simtext_app) %} with the following parameters:
+>    - {% icon param-file %} *"Input file"*: initial input file with genes and pre-existing grouping
+>    - {% icon param-file %} *"Matrix file"*: output of **PMIDs to PubTator** {% icon tool %}
+>
+> 2. Open interactive tool
```
Comment (shiltemann) [inline topics/statistics/tutorials/text-mining_simtext/tutorial.md:126]: Ah nice, you found the new snippet syntax :)

This step is probably pretty useful in more tutorials. Would you like to create a snippet (faq) for it? You can pass variables to snippets too likes so:

```
{% snippet faqs/galaxy/interactive_tools_open.md  tool="SimText App" %}
```
and in the snippet itself:

```
..
- Go to User > Active InteractiveTools
- Click on {% include.tool %}
..
```

the tutorial looks really good, thanks @dlal-group :+1:

Comment (dlal-group) [reply topics/statistics/tutorials/text-mining_simtext/tutorial.md:126]: Hi @shiltemann and thanks!

I tried to create and integrate the faq by creating a md file in faqs/galaxy and adding the faq to the tutorial md file. However, I get an error when trying to load the page with jekyll. 

The error messages:

When I use the {% include.tool %} strategy you showed me, using tool="SimText app" or "tool="interactive_tool_simtext_app" I get this error:
`Liquid Exception: Could not locate the included file '.tool' in any of ["…../training-material/"]. Ensure it exists in one of those directories and, if it is a symlink, does not point outside your site source. in faqs/galaxy/interactive_tools_open.md`

When I tried to use the faq without {% include.tool %}  and "tool=.." I get this error:
`Liquid Exception: (<unknown>): could not find expected ':' while scanning a simple key at line 6 column 1 in topics/statistics/tutorials/text-mining_simtext/tutorial.md
`

Could it be that another file is needed? I can see for example that all faqs also have an html file under _site/training-material/faqs.

Thank you for your help!

Comment (hexylena) [reply topics/statistics/tutorials/text-mining_simtext/tutorial.md:126]: Try `{{ include.tool }}` and that will probably help :)

Comment (shiltemann) [reply topics/statistics/tutorials/text-mining_simtext/tutorial.md:126]: ah, my bad, I messed up the example, yes, the double braces like helena points out should work

Comment (dlal-group) [reply topics/statistics/tutorials/text-mining_simtext/tutorial.md:126]: thank you, it works!:)

Comment (dlal-group) [reply topics/statistics/tutorials/text-mining_simtext/tutorial.md:126]: @hexylena @shiltemann I am submitting a review of a paper on Wednesday and would love to mention that there is a tutorial available. Is there anything else I should change? And are we waiting for other reviewers or is one of you able to merge this PR? I appreciate your help, thank you!

Comment (bgruening) [reply topics/statistics/tutorials/text-mining_simtext/tutorial.md:126]: I will take a look today!

Comment (dlal-group) [reply topics/statistics/tutorials/text-mining_simtext/tutorial.md:126]: Thank you @bgruening !

**Review by bgruening — APPROVED** [review]

**Comment by bgruening** [comment]
Thanks @dlal-group! Good luck with your paper. 
Thanks @shiltemann @hexylena for the review!

**Pull request merged**